opta/module_processors/aws_k8s_service.py

```diff
@@ -96,12 +96,25 @@ def process(self, module_idx: int) -> None:
         super(AwsK8sServiceProcessor, self).process(module_idx)
 
     def handle_rds_link(
-        self, linked_module: "Module", link_permissions: List[str]
+        self, linked_module: "Module", link_permissions: List[Any]
     ) -> None:
-        for key in ["db_user", "db_name", "db_password", "db_host"]:
+        required_db_vars = ["db_user", "db_name", "db_password", "db_host"]
+        renamed_vars = {}
+        if len(link_permissions) > 0:
```

Comment:
Can we pull this out in a common function? instead of copy pasting everywhere? A lot of this linking code could be pulled out I think.

Reply:
I was thinking about this -- my hesitation was in keeping informative error messages custom to each resource

Reply:
The error messages seem pretty similar and some loss in customization there seems worth it - for code clarity. Just my 2c. Totally fine to skip for now!